docs/Changelog.md

```diff
@@ -27,6 +27,7 @@ Nussknacker versions
 * [#1373](https://github.com/TouK/nussknacker/pull/1373) Ability to load custom model config programmatically
 * [#1406](https://github.com/TouK/nussknacker/pull/1406) Eager services - ability to create service object using static parameters
 * [#1428](https://gihub.com/TouK/nussknacker/pull/1428) Kafka SchemaRegistry source/sink can use JSON payloads. In this PR we assume one schema registry contains either json or avro payloads but not both.                                         
+* [#1439](https://github.com/TouK/nussknacker/pull/1439) Upgrade do Flink 1.12          
```

Comment:
Migration guide is not needed? What dev should do with missing `UserClassLoader`? what about new way of dealing with timestamps?

Reply:
Good point, I added guide